client/layout/masterbar/logged-out.jsx

```diff
@@ -13,7 +13,7 @@ import { localize } from 'i18n-calypso';
 import Item from './item';
 import config from 'config';
 
-const MasterbarLoggedOut = ( { title, translate } ) => (
+const MasterbarLoggedOut = ( { title, translate, showSignup = true } ) => (
```

Comment:
Let's add `showSignup` to `.propTypes` in L44, and although I know they achieve the same thing, we should probably either define the default in `defaultProps` (L48 - like `title`), and remove it here, or define `title = ''` here, and remove `.defaultProps` for consistency.